### DISCUSSION TITLE
No ckpt file in Drive

### DISCUSSION THREAD

**Discussion by benm11** [TheLastBen/fast-stable-diffusion#865] [General]
Hello,

FYI I followed all steps of the following tutorial: https://www.youtube.com/watch?v=ravETUa84P8 
I ran the whole notebook and have a 'session' folder in my GDrive, but I can't see any ckpt file. In the notebook, it said that the ckpt file was created and stored in the Drive though.

Is it possible that I didn't have enough space on the Drive? How much free GB should I have ? Do I have to rerun everything?

Thanks

**Comment by SAC020** [comment]
It takes a few minutes after the "done" notification for the CKPT to actually appear in the session folder (not sure why, but this is what happens)

**Reply by benm11** [reply]
It's been a day now

**Reply by TheLastBen** [reply]
there must be an error in training or you don't have enough space in your gdrive
use the latest colab https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb and try a training a model for only 10 steps and see if the CKPT is stored in your gdrive

**Reply by benm11** [reply]
I think it's because I only had 1GB of space left. How much space should I free up? I use 32 instance images

**Reply by TheLastBen** [reply]
the model is 2GB if fp16 and 4GB if not. make sure you have at least 5GB free to avoid any issues
